spec/features/admin_orders_view_spec.rb

```diff
@@ -6,11 +6,12 @@
     @user = FactoryGirl.create(:user)
     @status = FactoryGirl.create(:status)
     @order = FactoryGirl.create(:order, user_id: @user.id, status_id: @status.id)
-    # @order2 = FactoryGirl.create(:order, status: @status)
-    visit login_path
-    fill_in("session_email", with: @admin.email)
-    fill_in("session_password", with: @admin.password)
-    click_button 'Sign in'
+    visit root_path
+    within("#signInModal") do
+      fill_in("session_email", with: @admin.email)
+      fill_in("session_password", with: @admin.password)
+      click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.